src/main/java/leduc/command/TodoCommand.java

```diff
@@ -33,26 +35,40 @@ public  TodoCommand(String user){
      * @param storage leduc.storage.Storage which deals with loading tasks from the file and saving tasks in the file.
      * @throws EmptyTodoException Exception caught when the description of the todo list is not given by the user.
      * @throws FileException Exception caught when the file can't be open or read or modify
+     * @throws PrioritizeLimitException Exception caught when the new priority is greater than 9 or less than 0.
      */
-    public void execute(TaskList tasks, Ui ui, Storage storage) throws EmptyTodoException, FileException {
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws EmptyTodoException, FileException, PrioritizeLimitException {
         String userSubstring;
         if(callByShortcut){
             userSubstring = user.substring(TodoCommand.todoShortcut.length());
         }
         else {
             userSubstring = user.substring(4);
         }
-        if (userSubstring.isBlank()) {
-            throw new EmptyTodoException();
+        String[] prioritySplit = userSubstring.split("prio");
+        TodoTask newTask = null;
+        int priority = -1 ;
+        if (prioritySplit.length != 1 && prioritySplit[1].trim().matches("\\d+")){
+            priority = Integer.parseInt(prioritySplit[1].trim());
+            if (priority < 0 || priority > 9) {
+                throw new PrioritizeLimitException();
+            }
+            if (prioritySplit[0].isBlank()){
+                throw new EmptyTodoException();
+            }
+            newTask = new TodoTask(prioritySplit[0],"[✗]",priority);
```

Comment:
Same as before, why the [✗]

Reply:
These constructors for different Tasks are already used in storage. It is to avoid to create an other constructor or to modify the first constructor and I will have to create a task by input the standard priority 5 in each instantiation in hard.

Do you think it is better to create another constructor or modify the first one ?

Reply:
There are no problem about creating another constructor